SQL/New_patches/2021-09-22-ConsentModule.sql

```diff
@@ -0,0 +1,53 @@
+-- Create module
+INSERT INTO modules (Name, Active) VALUES ('consent', 'Y');
+
+-- Table for display information
+-- Can contain basic display info, or more complex
+-- training form
+CREATE TABLE `consent_display` (
+  `ConsentDisplayID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `Title` text DEFAULT NULL,
+  `Media` text DEFAULT NULL,
+  `Description` text DEFAULT NULL,
+  `training` text DEFAULT NULL,
+  `Reset_period_days` int(10) unsigned DEFAULT NULL,
+  PRIMARY KEY (`ConsentDisplayID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- direct_consent contains all info relevant to the eConsent
+-- form unique to each participant, but does not include the
+-- consent status given
+CREATE TABLE `direct_consent` (
+  `CandidateID` int(6) NOT NULL,
+  `ConsentGroupID` integer unsigned NOT NULL,
```

Comment:
No I don't think we discussed that option

Reply:
Just thought about this a bit more. I'm not sure that candidate_consent_rel is the best place to have a DirectConsentID since entries in the candidate_consent_rel are not necessarily eConsent related, and also the table is not organized by consent group. I feel like it complicates the use of the tables

Reply:
Direct ConsentID here would be for the direct_consent table, not the candidate_consent_rel table